app/assets/stylesheets/components/modals.scss

```diff
@@ -2,7 +2,7 @@
 
 .crayons-modal {
   --modal-max-width: 768px;
-  --modal-body-padding: var(--su-2);
+  --modal-body-padding: var(--su-3);
```

Comment:
Since this part of the modal itself is not being used anywhere else in DEV, I've decided to include it in this PR since it is actually needed there too: it addresses overflow issue @benhalpern has mentioned in earlier comments.

Reply:
Isn't the modal being used in onboarding?

Reply:
Parts of it, yes. But I have tested it the onboarding after this change ✅